frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentSettingsSection.tsx

```diff
@@ -39,20 +39,47 @@ import {
   StripeConnectButtonStates,
 } from './StripeConnectButton'
 
-const PaymentsNotAllowedMessage = () => {
-  return (
-    <Box w="100%">
-      <InlineMessage>
-        <Text>
-          To enable payment fields, remove all recipients from{' '}
-          <Link as={ReactLink} to={'email-notifications'}>
-            email notifications
-          </Link>
-          .
-        </Text>
-      </InlineMessage>
-    </Box>
-  )
+const getPaymentsNotAllowedMessageText = (
+  isEmailsPresent: boolean,
+  isSingleSubmission: boolean,
+) => {
+  if (isEmailsPresent && isSingleSubmission) {
+    return (
+      <Text>
+        To enable payment fields, remove all recipients from{' '}
+        <Link as={ReactLink} to={'email-notifications'}>
+          email notifications
+        </Link>{' '}
+        and disable{' '}
+        <Link as={ReactLink} to={'singpass'}>
+          only one submission per NRIC/FIN
+        </Link>
+        .
+      </Text>
+    )
+  } else if (isEmailsPresent) {
```

Comment:
/nit I generally avoid using `else if`s, and this particular block of code is already written in the style of a guard statement. We can be clearer by keeping it as `if`s with `return`s after every case.


```
if (ans % 100) {
  return (
    <>
      <Food />
    </>
  )
}
if (ans % 50) {
  return (
    <>
      <Cats />
    </>
  )
}
if (ans % 5) {
  return (
    <>
      <Dogs />
    </>
  )
}
return null
```

Reply:
yeah, this works! fixed!